src/script/opcodes/constant.zig

```diff
@@ -391,3 +391,27 @@ pub fn isUnnamedPushNDataOpcode(opcode: Opcode) ?u8 {
     const opcodeByte = opcode.toBytes();
     return if (opcodeByte > 0x00 and opcodeByte <= 0x4b) opcodeByte else null;
 }
+
+/// Conditional execution constants for script operations.
+pub const OpConditional = enum(u8) {
+    /// Represents a false condition (0)
+    False = 0,
+
+    /// Represents a true condition (1)
+    True = 1,
+
+    /// Represents a condition to skip (2)
+    Skip = 2,
+
+    /// Convert the enum to its underlying u8 value
+    pub fn toU8(self: OpConditional) u8 {
+        return @intFromEnum(self);
+    }
+
+    /// Create an OpConditional from a u8 value
+    pub fn fromU8(value: u8) !OpConditional {
+        return std.meta.intToEnum(OpConditional, value) catch {
+            return error.InvalidOpConditional;
+        };
+    }
```

Comment:
By using the @tdelabro method to include `CondStackValue` as a data type as an alternative to `u8`, you will not need to use these functions.

Reply:
@KhairallahA if I don't make a PR in the next 24hrs, you can take over this. I am still trying to get settled and not had enough time to dig into it so far.

cc: @tdelabro

Reply:
@okhaimie-dev I can feel that, brother! Much respect to you and your great effort in trying to settle down with such hard work. I’ve been there years ago. I have no problem doing this to avoid wasting more time on the project. And I hope that everything I'll execute and publish for this is considered as you the main one who made the attempt and did it. The unexpected external pressures are not optional for any person.

let me know if you have any other issues that are out of scope, if I can help you please.

cc: @tdelabro